docs/how-to-use-the-solution/README.md

```diff
@@ -708,6 +709,23 @@ It is necessary to have a [scheduled meeting](https://support.microsoft.com/en-u
         }
     }
     ```
+### Generate RTMP Stream Key
+1. **Generate RTMP Stream Key:** This endpoint updates the streamKey in the privateContext of the call, used key for RTMP  injections and extractions.
```

Comment:
**nitpick**: We have a double white-space between "RTMP" and "injections". Also lets add a sentence here saying that you only need to use the endpoint if you want to change the RMTP key of the current meeting, and that each meeting has its own RTMP key.